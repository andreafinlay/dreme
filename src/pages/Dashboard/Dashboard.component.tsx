import React, { useContext } from 'react';

import { RootContext } from '../../context/RootContext';
import { LoginForm, RegisterForm } from '../../components';

const Dashboard: React.FC<any> = () => {
    const { authenticated, name } = useContext(RootContext);
    return (
        <>
            {authenticated ? (
                <>
                    <h1>Hello, {name}</h1>
                </>
            ) : (
                <>
                    <RegisterForm />
                    <LoginForm />
                </>
            )}
        </>
    );
};

export { Dashboard };
