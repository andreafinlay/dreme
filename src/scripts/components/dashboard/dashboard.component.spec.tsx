import React from 'react';
import ReactDOM from 'react-dom';
import Dashboard from './dashboard.component';

xit('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Dashboard />, div);
    ReactDOM.unmountComponentAtNode(div);
});
