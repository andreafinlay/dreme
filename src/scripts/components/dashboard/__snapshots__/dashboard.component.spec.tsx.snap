// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`init should render correctly 1`] = `
<div
  className="bg-dreme"
>
  <header>
    Zzz
  </header>
  <UISref
    to="journal"
  >
    <a>
      Journal
    </a>
  </UISref>
</div>
`;
